taxonomies/packaging_materials.txt

```diff
@@ -256,6 +257,7 @@ hu:Alkáli elem, 19 Alkáli, 19
 nl:Alkalinebatterij
 nl_be:Alkalinebatterij
 pt:Bateria alcalina, pilha alcalina, 19 Alkaline, 19
+xx:19 Alkaline, 19
```

Comment:
```suggestion
xx:19 Alkaline
```

plain numbers should not be added as synonym, that would trigger many false positives